README.md

```diff
@@ -173,7 +176,7 @@ Describe what a schema is, and how we represent a one-to-many relationship in a
 SQL database. If you need an example, you can use `people` and `wishes` models.
 
 Your answer...
-
+The schema basically represents the table's columns.  It lays out what the columns consist of as well as the data types within those columns.
```

Comment:
This is good for the first half of the question concerning schema, but for db relationships we were looking for something about foreign keys or how (with the people/wishes example) the `wishes` table might have `person_id` column to show each with is related to a single `person` and a single person could have many wishes.